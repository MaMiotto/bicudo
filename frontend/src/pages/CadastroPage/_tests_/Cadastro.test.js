import { render, screen } from "@testing-library/react";
import { shallow } from "enzyme";
import Cadastro from "../cadastro"

describe("<Cadastro/>", () => {
    it("renders correctly", () => {
      const wrapper = shallow(<Cadastro />);
      expect(wrapper).toMatchSnapshot();
    });
});