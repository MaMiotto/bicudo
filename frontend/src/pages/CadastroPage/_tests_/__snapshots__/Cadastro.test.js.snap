// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cadastro/> renders correctly 1`] = `ShallowWrapper {}`;
